Dockerfile

```
@@ -9,12 +9,26 @@ RUN curl -fsSL https://bazel.build/bazel-release.pub.gpg | gpg --dearmor > bazel
     && echo "deb [arch=amd64] https://storage.googleapis.com/bazel-apt stable jdk1.8" | tee /etc/apt/sources.list.d/bazel.list \
     && apt update && apt install -y bazel
 
-WORKDIR /app
+RUN groupadd -g 1000 myuser && \
```

here and in all places
```suggestion
RUN groupadd -g 1000 user && \
```

---

fixed